src/components/Contacts/ContactDetail/ContactDetailTab/ContactDetailTab.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Button, Divider, styled, Typography } from '@material-ui/core';
+import { Create } from '@material-ui/icons';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ContactDetailQuery } from '../ContactDetail.generated';
+
+const ContactDetailSectionContainer = styled(Box)(({ theme }) => ({
+  padding: theme.spacing(0),
+  paddingTop: '4px',
+  margin: 0,
+  alignContent: 'start',
+}));
+
+const ContactDetailHeadingContainer = styled(Box)(() => ({
+  display: 'flex',
+  flexDirection: 'row',
+  alignContent: 'center',
+  alignItems: 'center',
+}));
+
+const ContactDetailHeadingIcon = styled(Create)(({ theme }) => ({
+  width: '18px',
+  height: '18px',
+  margin: theme.spacing(0),
+  color: '#9C9FA1',
+}));
+
+const ContactDeleteButton = styled(Button)(({ theme }) => ({
+  display: 'block',
+  padding: theme.spacing(2),
+  margin: theme.spacing(5),
+  marginLeft: 'auto',
+  marginRight: 'auto',
+}));
+
+const ContactDetailHeadingText = styled(Typography)(() => ({
+  flexGrow: 5,
+}));
+
+interface ContactDetailTabProps {
+  contact: ContactDetailQuery;
```

Comment:
What component is going to load this data? I feel like this component probably should. It could just take a `contactId` as a prop and query the GraphQL API.

Reply:
@OzzieOrca I was looking at @reldredge71  PR for the whole contact Detail Page and it looks like his ContactDetail fragment could replace this Query.  What do you think?

Reply:
Maybe @reldredge71  has thoughts based on how he's implementing the tabs.

It appears that tabs aren't rendered unless they are active so normal querying should work fine. https://github.com/mui-org/material-ui/blob/next/packages/material-ui-lab/src/TabPanel/TabPanel.js#L35

I think the contacts panel should have a query for the header/summary section and a query for each tab. Each of those components can be responsible for running their respective queries. I don't think there's any reason for these components to share data or move the query up to any parent components.

Since the tabs don't all load at once I don't think it makes sense to query this all at once. If it did we could use fragments but I'm not convinced we need one big query.

Guess there might be a few fields that would be loaded twice between the header and the contact details tab. I think it's fine to just fetch them twice, especially since the contact details isn't the default tab.

Reply:
@OzzieOrca sounds good I will make the changes.